index.test-d.ts

```diff
@@ -1,17 +1,22 @@
 import {expectType} from 'tsd';
 import camelcaseKeys = require('.');
 
-expectType<Array<{[key: string]: unknown}>>(camelcaseKeys([{'foo-bar': true}]));
-expectType<{[key: string]: unknown}>(camelcaseKeys({'foo-bar': true}));
-expectType<{[key: string]: unknown}>(
-	camelcaseKeys({'foo-bar': true}, {deep: true})
+expectType<Array<{[key in 'foo-bar']: true}>>(camelcaseKeys([{'foo-bar': true}]));
```

Comment:
Can you add a test where the expected type is the same object as passed to camelcaseKeys?

Reply:
@sindresorhus Do you mean something along the lines of...?:

```
expectType<Array<{'foo-bar': true}>>(camelcaseKeys([{'foo-bar': true}]));
```

Reply:
No, move the object to a variable and pass it in to both the expected type and camel case keys.

Reply:
@sindresorhus Additional tests pushed